home-service/src/main/java/com/softserveinc/ita/homeproject/service/UserService.java

```diff
@@ -0,0 +1,20 @@
+package com.softserveinc.ita.homeproject.service;
+
+import com.softserveinc.ita.homeproject.model.CreateUser;
+import com.softserveinc.ita.homeproject.model.ReadUser;
+import com.softserveinc.ita.homeproject.model.UpdateUser;
+
+import java.util.Collection;
+
+public interface UserService {
+
+    ReadUser createUser(CreateUser payload);
+
+    ReadUser updateUser(Long id, UpdateUser payload);
```

Comment:
avoid such naming - `payload`. Would be better `UpdateUserDto updateUserDto`

Reply:
Noted, will be fixed